egs/wsj/s5/local/chain/tuning/run_tdnn_1g.sh

```diff
@@ -46,7 +53,7 @@ xent_regularize=0.1
 dropout_schedule='0,0@0.20,0.5@0.50,0'
 
 # training chunk-options
-chunk_width=140,100,160
+chunk_width=140,100,60
```

Comment:
can you not change the chunk_width?  This could affect the results.

Reply:
I was written in documentation, that the chunk-width should be a list of integers in descending order (try length 140, 100, ...), therefore the 160 did not make sense to me, which is why I changed it. Or am I wrong? What was the intention of using the values '140,100,160' ?